src/gui/MainWindow.cpp

```diff
@@ -607,6 +607,34 @@ MainWindow::MainWindow()
     }
 #endif
 
+    if(osUtils->canPreventScreenCapture()) {
+        if(osUtils->allowScreenCapture()
+            && not m_ui->globalMessageWidget->text().isEmpty()) {
+            m_ui->globalMessageWidget->hideMessage();
+            m_ui->globalMessageWidget->showMessage(
+                m_ui->globalMessageWidget->text(),
+                m_ui->globalMessageWidget->messageType()
+            );
+        }
+        connect(m_ui->globalMessageWidget, &MessageWidget::hideAnimationFinished, [this](){
+            if (osUtils->allowScreenCapture()) {
+                m_ui->globalMessageWidget->showMessage(
+                    tr("WARNING: Application screen capture is allowed!\n"),
+                    MessageWidget::Warning,
+                    -1
+                );
+            }
+        });
+        connect(this, &MainWindow::windowTitleChanged, [this](const QString& title){
+            if (osUtils->allowScreenCapture()) {
+                static const auto postfix = tr(" [Screen Capture Allowed]", "MainWindow");
+                if (!title.contains(postfix)) {
+                    setWindowTitle(title + postfix);
+                }
+            }
+        });
+    }
+
```

Comment:
Please remove all of this, it is not necessary and will be incredibly annoying while debugging or doing documentation work. The user has to very explicitly add a command line flag to enable screenshots, that in and of itself is enough barrier and warning.

Reply:
Done